extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -238,15 +237,19 @@ export class DraggableDrawer extends AMP.BaseElement {
    * @private
    */
   startListeningForTouchEvents_() {
-    // If the element is a direct descendant of amp-story-page, authorize
-    // swiping up by listening to events at the page level. Otherwise, only
-    // authorize swiping down to close by listening to events at the current
-    // element level.
+    // If the element is a direct descendant of amp-story-page or a decendent
+    // of amp-story-shopping-attachment, authorize swiping up by listening to
+    // events at the page level. Otherwise, only authorize swiping down to
+    // close by listening to events at the current element level.
     const parentEl = this.element.parentElement;
-    const el = dev().assertElement(
+    let el = dev().assertElement(
```

Comment:
Nit: Mind renaming `el` to `targetEl`, just so it's more clear that it represents the target of these touch events? `el` is a bit vague imo